source/fuzz/transformation_equation_instruction.cpp

```diff
@@ -38,10 +38,11 @@ TransformationEquationInstruction::TransformationEquationInstruction(
 
 bool TransformationEquationInstruction::IsApplicable(
     opt::IRContext* ir_context, const TransformationContext& /*unused*/) const {
-  // The result id must be fresh.
+  // |fresh_id| must all be fresh.
```

Comment:
Revert back to singular as there is only one of them.

Reply:
Fixed in a6ad6b3.